### PR TITLE
(tests) Added tests for appointment metrics

### PR DESCRIPTION
**Pull request by CynthiaKamau**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary
- Added tests for appointment metrics and added mock data

## Screenshots

*None.*

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

File: packages/esm-appointments-app/src/appointments-metrics/appointments-metrics.test.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    useClinicalMetrics: jest.fn().mockImplementation(() => ({
+      totalAppointments: mockAppointmentMetrics.totalAppointments,
+      highestServiceLoad: mockAppointmentMetrics.highestServiceLoad,
+      isLoading: mockAppointmentMetrics.isLoading,
+      error: mockAppointmentMetrics.error,
+    })),
+  };
+});
+
+jest.mock('@openmrs/esm-framework', () => {
```
Comment (denniskigen) [inline packages/esm-appointments-app/src/appointments-metrics/appointments-metrics.test.tsx:23]: It looks like we're not mocking this module - best to remove the mock altogether.

File: packages/esm-appointments-app/src/appointments-metrics/appointments-metrics.test.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  const originalModule = jest.requireActual('@openmrs/esm-framework');
+
+  return {
+    ...originalModule,
+  };
+});
+
+describe('Appointment metrics', () => {
+  it('renders metrics from appointment list', async () => {
```
Comment (denniskigen) [inline packages/esm-appointments-app/src/appointments-metrics/appointments-metrics.test.tsx:32]: Also appears like we're not doing anything asynchronous in the test body, so we wouldn't need this either.

File: packages/esm-appointments-app/src/appointments-metrics/appointments-metrics.test.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    mockedOpenmrsFetch.mockResolvedValueOnce({ data: mockAppointmentMetrics });
```
Comment (denniskigen) [inline packages/esm-appointments-app/src/appointments-metrics/appointments-metrics.test.tsx:33]:
```suggestion
    mockedOpenmrsFetch.mockResolvedValue({ data: mockAppointmentMetrics });
```

File: packages/esm-appointments-app/src/appointments-metrics/appointments-metrics.test.tsx (hunk 1 above)
Comment (CynthiaKamau) [reply packages/esm-appointments-app/src/appointments-metrics/appointments-metrics.test.tsx]: Done

**Review by denniskigen — APPROVED** [review]

**Comment by denniskigen** [comment]
Thanks, @CynthiaKamau.

**Pull request merged**